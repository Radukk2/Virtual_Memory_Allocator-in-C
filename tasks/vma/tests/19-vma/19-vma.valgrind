==7017== Memcheck, a memory error detector
==7017== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==7017== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==7017== Command: /home/student/Desktop/SDA/vma
==7017== Parent PID: 7016
==7017== 
==7017== 
==7017== HEAP SUMMARY:
==7017==     in use at exit: 0 bytes in 0 blocks
==7017==   total heap usage: 88 allocs, 88 frees, 10,325 bytes allocated
==7017== 
==7017== All heap blocks were freed -- no leaks are possible
==7017== 
==7017== For lists of detected and suppressed errors, rerun with: -s
==7017== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
