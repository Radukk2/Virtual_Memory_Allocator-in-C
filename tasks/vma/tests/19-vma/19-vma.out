Total memory: 0x14 bytes
Free memory: 0x10 bytes
Number of allocated blocks: 1
Number of allocated miniblocks: 4

Block 1 begin
Zone: 0x3 - 0x7
Miniblock 1:		0x3		-		0x4		| RW-
Miniblock 2:		0x4		-		0x5		| RW-
Miniblock 3:		0x5		-		0x6		| RW-
Miniblock 4:		0x6		-		0x7		| RW-
Block 1 end
Total memory: 0x14 bytes
Free memory: 0xE bytes
Number of allocated blocks: 2
Number of allocated miniblocks: 3

Block 1 begin
Zone: 0x3 - 0x7
Miniblock 1:		0x3		-		0x4		| RW-
Block 1 end

Block 2 begin
Zone: 0x5 - 0x7
Miniblock 1:		0x5		-		0x6		| RW-
Miniblock 2:		0x6		-		0x7		| RW-
Block 2 end
This zone was already allocated.
Total memory: 0x14 bytes
Free memory: 0xE bytes
Number of allocated blocks: 2
Number of allocated miniblocks: 3

Block 1 begin
Zone: 0x3 - 0x7
Miniblock 1:		0x3		-		0x4		| RW-
Block 1 end

Block 2 begin
Zone: 0x5 - 0x7
Miniblock 1:		0x5		-		0x6		| RW-
Miniblock 2:		0x6		-		0x7		| RW-
Block 2 end
Invalid address for free.
Total memory: 0x14 bytes
Free memory: 0xE bytes
Number of allocated blocks: 2
Number of allocated miniblocks: 3

Block 1 begin
Zone: 0x3 - 0x7
Miniblock 1:		0x3		-		0x4		| RW-
Block 1 end

Block 2 begin
Zone: 0x5 - 0x7
Miniblock 1:		0x5		-		0x6		| RW-
Miniblock 2:		0x6		-		0x7		| RW-
Block 2 end
